CoroutineTemplate/domain/src/main/java/co/nimblehq/coroutine/domain/usecase/GetUsersUseCase.kt

```diff
@@ -1,24 +0,0 @@
-package co.nimblehq.coroutine.domain.usecase
-
-import co.nimblehq.coroutine.domain.data.entity.UserEntity
-import co.nimblehq.coroutine.domain.dispatcher.IoDispatcher
-import co.nimblehq.coroutine.domain.repository.UserRepository
-import kotlinx.coroutines.CoroutineDispatcher
-import kotlinx.coroutines.withContext
-import javax.inject.Inject
-
-class GetUsersUseCase @Inject constructor(
-    private val userRepository: UserRepository,
-    @IoDispatcher private val ioDispatcher: CoroutineDispatcher
-) {
-    suspend fun execute(): UseCaseResult<List<UserEntity>> {
```

Comment:
originally, it doesn't have any pagination argument 🤔

Reply:
@sleepylee Yep, I thought I had, but not. I will update by removing the pagination argument.